src/pkcs7.rs

```diff
@@ -51,15 +74,20 @@ pub mod pkcs7 {
         #[test]
         fn test_unpad_single_byte() {
             let input = vec![0, 0, 0, 1];
-            let result = unpad(&input);
-            assert_eq!(vec![0, 0, 0], result);
+            match unpad(&input, 4) {
+                Ok(v) => assert_eq!(vec![0, 0, 0], v),
+                Err(e) => panic!("{}", e)
+            }
```

Comment:
In the successful assertion instances I think we can write it as:

```rust
let result = unpad(&input).unwrap();
assert_eq!(vec![0, 0, 0], result);
```

But given the guidance [here](https://doc.rust-lang.org/std/result/enum.Result.html#method.unwrap) I'm learning its only if you _want_ a runtime panic as a result of `Err`. I'm probably using `unwrap` elsewhere where it shouldn't be.

Reply:
Ah, so that's what the unwrap() is doing, that makes a lot more sense now.